docs/QTIP.md

```diff
@@ -0,0 +1,25 @@
+# QTIP
+
+## What?
+
+QTIP is an MsQuic specific concept that allows MsQuic to exercise the QUIC protocol over XDP using TCP packets.
```

Comment:
Not necessarily for that PR, but since QTIP is MsQuic specific, we should explain it more in detail somewhere.
I am not familiar with the details, but IIUC, there is more to it that just sending Quic packets over TCP, and is closer to sticking a "fake" TCP header over QUIC packets?

Reply:
Yes, I originally had the paragraph "We put the QUIC packet inside the TCP payload, and have fake data in the TCP header, just real enough to fool the environment"... but I thought that went too much into detail. I can add that back in.